product_docs/docs/pgd/5/planning/limitations.mdx

```diff
@@ -78,9 +78,14 @@ and current listing.
 
 ## Mixed PGD versions
 
-PGD was developed to [enable rolling upgrades of PGD](/pgd/latest/upgrades) by allowing mixed versions of PGD to operate during the upgrade process. 
-We expect users to run mixed versions only during upgrades and, once an upgrade starts, that they complete that upgrade.
-We don't support running mixed versions of PGD except during an upgrade.
+PGD was developed to [enable rolling upgrades of PGD](/pgd/latest/upgrades) by
```

Comment:
@djw-m need to adjust this to explain more

One of PGD's key value propositions is in its ability to enable rolling upgrades, allowing users to avoid downtime associated with moving to new versions. We recommend testing rolling upgrades in a test environment before going to production. When performing a rolling upgrade, users add a new node to an existing cluster, which results in a mixed version cluster. We recommend performing rolling upgrades during off-peak hours and completing the process in one go in order to minimize the time the cluster is in a mixed version state. You can't use the new features in the new version until the rolling upgrade is complete, and the node with the old version has been removed. PGD does not support mixed version clusters for production. As with any mixed version software, there are common pitfalls that might arise if the upgrade is not completed including but not limited to: incompatibility issues between the different versions causing errors and inefficiencies and increased complexity during troubleshooting due to version discrepancies. 

While performing a rolling upgrade, you cannot change database types i.e. from Postgres to EDB Postgres Extended.

Reply:
This is the Limitations section where we are historically brief on the issue so that the explicit limitation is directly visible. If anything, this section has expanded to the point of being too large as we don't have an explicit limitation on the use on mixed versions.